### DISCUSSION TITLE
Camera zoom sensitivity

### DISCUSSION THREAD

**Discussion by wawa2016** [google/model-viewer#1969] [Q&A] [answered]
I have a large terrain model 🗺️ and when zooming with scroll wheel the camera jumps are too large. How can I make them more incremental? Is there a zoom sensitivity setting for the camera or do I need to adjust the scale of my model?

**Comment by elalish** [comment, marked as answer]
This is tough because different mouse and OS settings can change this sensitivity already, so consistency is all but impossible. If you make the `min-field-of-view` closer to `max-field-of-view` or the min orbit radius closer to the max orbit radius, then you won't be able to zoom as far, which will also make it less sensitive.
